cmd/loopd/config.go

```diff
@@ -1,5 +1,25 @@
 package main
 
+import (
+	"path/filepath"
+
+	"github.com/btcsuite/btcutil"
+)
+
+var (
+	loopDirBase = btcutil.AppDataDir("loop", false)
+
+	defaultConfigFilename = "loopd.conf"
+
+	defaultLogLevel    = "info"
+	defaultLogDirname  = "logs"
+	defaultLogFilename = "loopd.log"
+	defaultLogDir      = filepath.Join(loopDirBase, defaultLogDirname)
```

Comment:
This creates the network directory level in two locations. There is `~/.loop/logs/regtest` for the logs and also `~/.loop/regtest` for the db. I think it is better to keep all files for a network underneath the same dir.